encrypted-media-respec.html

```diff
@@ -2088,35 +2086,34 @@
             <li><p>Set the <var>session</var>'s <a def-id="expiration"></a> attribute to <var>expiration time</var>.</p></li>
           </ol>
         </section>
-
-        <section id="session-close">
-          <h4>Session Close</h4>
-          <p>The Session Close algorithm is run when the CDM closes the session associated with a <a>MediaKeySession</a> object.</p>
-          <p>Closing a session means that the license(s) and key(s) associated with it are no longer available to decrypt <a def-id="media-data"></a>.
-            All <a>MediaKeySession</a> methods will fail for this object.
+      
+        <section id="session-closed">
+          <h4>Session Closed</h4>
+          <p>The Session Closed algorithm updates the <a>MediaKeySession</a> state after a <a href="#key-session">session</a> has been closed.</p>
+          <p>
+            Closing a <a href="#key-session">session</a> means that the license(s) and key(s) associated with it are no longer available to
+            decrypt <a def-id="media-data"></a>. All <a>MediaKeySession</a> methods will fail for this object after this algorithm is run.
           </p>
-          <p class="note">The CDM may close a session at any point, such as in response to a <a def-id="close"></a> call, when the session is no longer needed, or when system resources are lost.
-          Keys in other sessions SHOULD be unaffected, even if they have overlapping key IDs.
+          <div class="note">
+            <p>
+              The CDM may close a session at any point, such as when the session is no longer needed or when system resources are lost.
+              In that case, this algorithm MUST be run.
+            </p>
```

Comment:
[Hmm, I thought I posted this comment already - apologies for duplicates]

I will add a note.

I assume the busy loop algorithm runs in the same "thread" (the one-and-only thread in which web platform specification algorithms execute) and so that algorithm cannot be used to send a message or anything because it only runs for as long as the session is not closed.